src/components/CreateArticle.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { createArticle } from '../actions/articleAction';
+import { createArticle, updateArticle } from '../actions/articleAction';
```

Comment:
Export all actions from the same file so you can have one line for the imports
`import { addFlashMessage , createArticle, updateArticle } from '../actions';`